src/components/molecules/LogInLinkGoogle.tsx

```diff
@@ -18,14 +15,17 @@ const useStyles = makeStyles({
     cursor: 'pointer',
   },
 });
+
 const LogInLinkGoogle = () => {
   const classes = useStyles();
   const handleClick = () => {
```

Comment:
do we still need handleClick here and in line 25?

Reply:
t was changed to our button instead of the `<a>` element.